### PR TITLE
Add keyten kt60-M 

### PR DESCRIPTION
**Pull request by key10iq**
## Description
Add keyten kt60-M PCB to QMK. It is a QMK and VIA-compatible 60% PCB with a minimum number of required layouts and cutouts under stabilizers with which sound at bottomout is a little quieter than without them. 
PCB is distributed through GBs hosted by me on [r/mm](https://www.reddit.com/r/mechmarket/comments/s4jvsc/gb_kt60m_60_universal_pcb_with_a_minimum_number).
Also this PR contains a new vendor folder (keyten) for all my works, since I have 2 more finished PCBs finished and 2 PCBs in the design stage right now.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Added new vendor folder - keyten
* Added new keyboard - keyten/kt60_m (QMK and VIA)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keyten/kt60_m/rules.mk [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/keyten/kt60_m/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/keyten/kt60_m/rules.mk [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/keyten/kt60_m/rules.mk:15]:
```suggestion
NKRO_ENABLE = yes           # Enable N-key Rollover
```

File: keyboards/keyten/kt60_m/rules.mk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/keyten/kt60_m/rules.mk:18]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by key10iq — COMMENTED** [review]
changed

File: keyboards/keyten/kt60_m/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# keyten kt60-M
+
+60% MX PCB with a minimum number of required layouts.
+
+![kt60-M image](https://i.imgur.com/ZAdGKmQ.jpg)
```
Comment (drashna) [inline keyboards/keyten/kt60_m/readme.md:5]:
```suggestion
![kt60-M image](https://i.imgur.com/ZAdGKmQl.jpg)
```

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by epiciskandar** [comment]
@key10iq it looks like `kt60_m.c` has insufficient line endings(\r\n), causes my git automatically changed them whenever trying to sync or reset.

**Comment by key10iq** [comment]
> @key10iq it looks like `kt60_m.c` has insufficient line endings(\r\n), causes my git automatically changed them whenever trying to sync or reset.

thank you for your remark! also thanks tzarc for fixing my mistake.

**Comment by fauxpark** [comment]
For future reference, if Git gets stuck on files like this, you can simply delete them and try updating the repo again after they've been fixed upstream.

**Pull request merged**